templates/jaasai/index.html

```diff
@@ -225,65 +225,47 @@ <h2>Why use Juju?</h2>
-        <div>
-          <div class="p-heading-icon">
-            <div class="p-heading-icon__header">
-                <img class="p-heading-icon__img" src="/static/img/logos/github-logo.svg" alt="Github logo">
-                <h3 class="p-heading-icon__title">
-                  <a href="https://github.com/juju">
-                    Juju on Github
-                  </a>
-                </h3>
-            </div>
-          </div>
-          <div class="github-actions">
-            <a class="github-actions__cta github-button" href="https://github.com/juju/juju" data-icon="octicon-star" data-style="mega" data-count-href="/juju/juju/stargazers" data-count-api="/repos/juju/juju#stargazers_count" data-count-aria-label="# stargazers on GitHub" aria-label="Star juju/juju on GitHub">Star</a>
-            <a class="github-actions__cta github-button" href="https://github.com/juju/juju/subscription" data-icon="octicon-eye" data-style="mega" data-count-href="/juju/juju/watchers" data-count-api="/repos/juju/juju#subscribers_count" data-count-aria-label="# watchers on GitHub" aria-label="Watch juju/juju on GitHub">Watch</a>
-            <a class="github-actions__cta github-button" href="https://github.com/juju/juju/fork" data-icon="octicon-repo-forked" data-style="mega" data-count-href="/juju/juju/network" data-count-api="/repos/juju/juju#forks_count" data-count-aria-label="# forks on GitHub" aria-label="Fork juju/juju on GitHub">Fork</a>
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3>Connect</h3>
+      <p>Join the Juju community on our <a href="https://discourse.jujucharms.com">Discourse forum</a>.</p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3>Collaborate</h3>
+      <div>
+        <div class="p-heading-icon">
+          <div class="p-heading-icon__header">
+            <img class="p-heading-icon__img" src="/static/img/logos/github-logo.svg" alt="Github logo">
+            <h3 class="p-heading-icon__title">
+              <a href="https://github.com/juju">
+                Juju on Github
+              </a>
+            </h3>
           </div>
         </div>
-        <div>
-          <p><b>Ask the community</b></p>
-          <p>Find out more, get help or ask any questions at <a class="external external-alt" href="http://askubuntu.com/questions/tagged/juju">ask ubuntu</a>.</p>
-        </div>
-      </div>
-      <div class="col-4 p-divider__block">
-        <h3>Participate</h3>
-        <div id="js-events-feed">
+        <div class="github-actions">
+          <iframe allowtransparency="true" scrolling="no" frameborder="0" src="https://buttons.github.io/buttons.html#href=https%3A%2F%2Fgithub.com%2Fjuju%2Fjuju&amp;text=Star&amp;data-count-api=%2Frepos%2Fjuju%2Fjuju%23stargazers_count&amp;data-count-href=%2Fjuju%2Fjuju%2Fstargazers&amp;data-count-aria-label=%23%20stargazers%20on%20GitHub&amp;data-style=mega&amp;data-icon=octicon-star&amp;aria-label=Star%20juju%2Fjuju%20on%20GitHub"
```

Comment:
Does this have to be an iframe? Can we not just add static images? I'm not a fan of allowing a third party to effectively update content (albeit their icon) on our homepage.